samples/subsys/display/lvgl/boards/mimxrt1060_evk_mimxrt1062_qspi_C.conf

```diff
@@ -0,0 +1,9 @@
+# Copyright 2023 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+# Enable PXP DMA engine and set rotation angle to 0 degrees.
```

Comment:
where is rotation angle set to 0?

Reply:
@kartben  this is reused from former platform, in PCP engine we can rotate the image by 90/180/270 degree, so this is default setting.

Reply:
This still needs to be addressed, imo. The comment is not needed if it is a default. It tends to confuse anyone who reads it, because nowhere is there any config about rotation in here.